include/xcp-ng/generic/string.h

```diff
@@ -29,6 +29,8 @@ XCP_NO_DISCARD int xcp_str_to_int (const char *str, bool *ok);
 XCP_NO_DISCARD long xcp_str_to_long (const char *str, bool *ok);
 XCP_NO_DISCARD longlong xcp_str_to_longlong (const char *str, bool *ok);
 
+char *xcp_str_trim_end (char *str);
```

Comment:
XCP_NO_DISCARD ?

Reply:
No useful. Because the returned pointer is always str.
`XCP_NO_DISCARD` must be used when the returned value is important and always necessary.